### PR TITLE
Renaming "./kubestr browse" command to "./kubestr browse pvc"

### PR DESCRIPTION
**Pull request by shlokc9**
This PR renames the `./kubestr browse` command to `./kubestr browse pvc` while maintaining the support for `./kubestr browse`.

Tested these code changes by executing both the commands on a GKE cluster. Please find the output below;

```
> ./kubestr browse pvc test-pvc1 -n file-level-restore -v test-snapshotclass
Taking a snapshot
Creating the file browser application.
Forwarding the port.
Port forwarding is ready to get traffic. visit http://localhost:8080/
^CStopping port forward
Cleaning up resources
``` 

```
> ./kubestr browse test-pvc1 -n file-level-restore -v test-snapshotclass
Taking a snapshot
Creating the file browser application.
Forwarding the port.
Port forwarding is ready to get traffic. visit http://localhost:8080/
^CStopping port forward
Cleaning up resources
```

During cleanup, we clean the placeholder VolumeSnapshot, PVC created from that VolumeSnapshot & the Filebrowser pod.

### CODE REVIEW

File: cmd/rootCmd.go [hunk 1]
```diff
@@ -83,9 +83,19 @@ var (
 		},
 	}
 
-	pvcBrowseLocalPort int
-	pvcBrowseCmd       = &cobra.Command{
-		Use:   "browse [PVC name]",
+	browseLocalPort int
+	browseCmd       = &cobra.Command{
```
Comment (bathina2) [inline cmd/rootCmd.go:87]: We should mark this command as deprecated and print a message to use `browse pvc` instead . 
https://github.com/spf13/cobra/blob/371ae25d2c82e519feb48c82d142e6a696fd06dd/command.go#L102

Comment (shlokc9) [reply cmd/rootCmd.go:87]: Done! ✅  This is how the `stdout` looks now.
```
> ./kubestr browse test-pvc1 -n file-level-restore -v test-snapshotclass
Command "browse" is deprecated, we recommend you to use command 'browse pvc' instead. Command 'browse pvc' will support newer updates for browsing through the contents of a PVC.
Taking a snapshot
Creating the file browser application.
Forwarding the port.
Port forwarding is ready to get traffic. visit http://localhost:8080/
^CStopping port forward
Cleaning up resources
```

File: cmd/rootCmd.go [hunk 2]
```diff
@@ -83,9 +83,20 @@ var (
...
+		Use:        "browse",
+		Short:      "Browse the contents of PVC or VolumeSnapshot",
+		Long:       "Browse the contents of a CSI provisioned PVC or a CSI provisioned VolumeSnapshot.",
+		Deprecated: "we recommend you to use command 'browse pvc' instead. Command 'browse pvc' will support newer updates for browsing through the contents of a PVC.",
```
Comment (bathina2) [inline cmd/rootCmd.go:91]:
```suggestion
		Deprecated: "We recommend you to use command 'browse pvc' instead. Command 'browse pvc' will support newer updates for browsing through the contents of a PVC.",
```

**Review by bathina2 — APPROVED** [review]
minor nits about messaging

File: cmd/rootCmd.go [hunk 3]
```diff
@@ -164,12 +175,19 @@ func init() {
...
+	browseCmd.Flags().StringVarP(&csiCheckVolumeSnapshotClass, "volumesnapshotclass", "v", "", "The name of a VolumeSnapshotClass. (Required)")
+	_ = browseCmd.MarkFlagRequired("volumesnapshotclass")
+	browseCmd.Flags().StringVarP(&namespace, "namespace", "n", fio.DefaultNS, "The namespace of the PersistentVolumeClaim.")
+	browseCmd.Flags().Int64VarP(&csiCheckRunAsUser, "runAsUser", "u", 0, "Runs the inspector pod as a user (int)")
+	browseCmd.Flags().IntVarP(&browseLocalPort, "localport", "l", 8080, "The local port to expose the inspector")
+
+	browseCmd.AddCommand(browsePvcCmd)
+	browsePvcCmd.Flags().StringVarP(&csiCheckVolumeSnapshotClass, "volumesnapshotclass", "v", "", "The name of a VolumeSnapshotClass. (Required)")
+	_ = browsePvcCmd.MarkFlagRequired("volumesnapshotclass")
+	browsePvcCmd.Flags().StringVarP(&namespace, "namespace", "n", fio.DefaultNS, "The namespace of the PersistentVolumeClaim.")
```
Comment (bathina2) [inline cmd/rootCmd.go:188]: make namespace a persistent flag

Comment (bathina2) [reply cmd/rootCmd.go:188]: As well as the `runAsUser` and the `localport` flags

**Pull request merged**